### DISCUSSION TITLE
Function to create a prior from uplift tests

### DISCUSSION THREAD

**Discussion by nialloulton** [pymc-labs/pymc-marketing#488] [Ideas]
When running uplift tests we get a resulting estimated impact of what the channel had as an impact, with additional potential insights on the adstock effect too, if we observe the experiment for long enough. Currently it's a manual process to translate uplift tests to priors. It would be great to translate this process into a function (or a demo example code in the documentation), which can translate outputs of uplift tests into priors. pm.find_contstrained_prior or similar type of approach is probably the way to go. 

Would love to hear opinions on ideal functionality for the user.

**Comment by CharliedT** [comment]
Amazing! It would be great to be able to translate a function that includes diminishing return unto that function

**Comment by BahgatN** [comment]
do you run a bayesian uplift tests? it would be nice to plug in the uplift tests posterior as a prior into mmm model somthething like `find_prior(posterior) `

**Comment by williambdean** [comment]
Working on this PR #590 in order to lift tests directly into the model. WIP and getting ready for review

The current format is that each lift test will have: 
- x: baseline x value
- delta_x: change in the x value
- delta_y: observed change in the y value from delta_x change in x
- sigma: confidence or standard deviation of the delta_y

**Comment by cetagostini** [comment]
Already available the PR #590 was merge!
